.ci_support/linux_64_.yaml

```
@@ -6,5 +6,7 @@ channel_targets:
 - conda-forge main
 docker_image:
 - quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.10'
```

I would suggest not doing this manually @orbeckst - otherwise it'll get overriden when you do a rebuild.

---

oops ... ok, then where does this need to be changed?

---

Ok, what's the alternative?

I am also don't quite understand why the Azure runner actually failed, given that there are versions of MDAnalysis >= 2.0.0 that should work with Python 3.9.

---

The standard approach is to use >=3.10 instead of python_min, will make a comment below.

---

Revert to bot's change:
```suggestion
- '3.9'
```